### PR TITLE
object detection plugin blogpost

### PR DESCRIPTION
**Pull request by nathakits**
draft new object detection blogpost

### CODE REVIEW

File: contents/articles/2017-12_object_fixation/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+--- 
+title: Real-time object recognition using fixations to control prosthetic hand
+date: Fri Dec 15 2017 11:31:00 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Real-time object recognition with fixations to control grasps of a robotic prosthetic hand..."
+featured_img: "../../../../media/images/blog/object_detection.jpg"
+featured_img_thumb: "../../../../media/images/blog/thumb/object_detection.jpg"
+---
+
+[Jesse Weisberg](https://github.com/jesseweisberg) has made real-time object detection plugin that integrated with Pupil Labs headset to indentify what object the user is currently fixated upon and control a prosthetic hand by forming grasps based on the fixated object.
```
Comment (willpatera) [inline contents/articles/2017-12_object_fixation/index.md]: edit: has made `a` real-time

Comment (willpatera) [inline contents/articles/2017-12_object_fixation/index.md]: Replace paragraph with below:

We are excited to share a project from a Pupil community member, [Jesse Weisberg]((https://github.com/jesseweisberg). Jesse has developed a plugin for Pupil that enables a prosthetic hand to form grasps based on what the wearer is fixating on in the scene in real-time on the CPU (10-15 fps)! 

This project is a great demonstration of how to extend Pupil and contribute back to the open source community!

File: contents/articles/2017-12_object_fixation/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+<div class="Feature-video-container-16by9">
+  <iframe class="Feature-video u-padTop--2" src="https://www.youtube.com/embed/MqqQnCbzryA?rel=0" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
```
Comment (willpatera) [inline contents/articles/2017-12_object_fixation/index.md]: change to the other video please.

File: contents/articles/2017-12_object_fixation/index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+</div>
+
+<div class="small u-padBottom--2">Robotic Prosthetic Hand Controlled with Object Recognition and Eye-Tracking</div>
+
+The object detection plugin publishes the detected object that is closest to the fixation to a rostopic. The Arduino subscribing to the rostopic, then controls the linear actuators of the prosthetic hand to form a predefined grasp associated with that object stored in the Arduino firmware.
```
Comment (willpatera) [inline contents/articles/2017-12_object_fixation/index.md]: `rostopic`. An Ardunio Mega 2560 subscribes to the rostopic and controls the linear actuators in an OpenBionics 3D printed hand to form a predefined grasp that is associated with the detected object.

File: contents/articles/2017-12_object_fixation/index.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+For more information on Jesse Weisberg's plugin, head over to his [github](https://github.com/jesseweisberg/pupil) project.
```
Comment (willpatera) [inline contents/articles/2017-12_object_fixation/index.md]: Check out Jesse's Pupil fork on [github](https://github.com/jesseweisberg/pupil). He provides a very well written readme along with demo videos. Or jump straight to his "object detector app" [in his repo](https://github.com/jesseweisberg/pupil/tree/master/pupil_src/shared_modules/object_detector_app). 

We hope that others will build on top of Jesse's work and contribute back to his project!

Comment (willpatera) [reply contents/articles/2017-12_object_fixation/index.md]: Finally, if you have built something with Pupil, please considering sharing with the community via chat on [DiscordApp](https://pupil-labs.com/chat).

File: contents/articles/2017-12_object_fixation/index.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+title: Real-time object recognition using fixations to control prosthetic hand
+date: Fri Dec 15 2017 11:31:00 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Real-time object recognition with fixations to control grasps of a robotic prosthetic hand..."
+featured_img: "../../../../media/images/blog/object_detection.jpg"
+featured_img_thumb: "../../../../media/images/blog/thumb/object_detection.jpg"
+---
+
+We are excited to share a project from a Pupil community member, [Jesse Weisberg]((https://github.com/jesseweisberg). Jesse has developed a plugin for Pupil that enables a prosthetic hand to form grasps based on what the wearer is fixating on in the scene in real-time on the CPU (10-15 fps)!
+
```
Comment (jesseweisberg) [inline contents/articles/2017-12_object_fixation/index.md]: Feel free to attach this to the end of previous paragraph or wherever you see fit.

This plugin was developed as part of a project called [Therabotics](https://www.jesseweisberg.com/therabotics/).  Therabotics was created by a team of graduate students at Washington University in St. Louis to explore how computer vision can be used to improve prosthetic hand control.

File: contents/articles/2017-12_object_fixation/index.md [hunk 6]
```diff
@@ -7,14 +7,20 @@ featured_img: "../../../../media/images/blog/object_detection.jpg"
...
+The object detection plugin publishes the detected object that is closest to the fixation to a rostopic. An Ardunio Mega 2560 subscribes to the rostopic and controls the linear actuators in an OpenBionics 3D printed hand to form a predefined grasp that is associated with the detected object.
+
+Check out Jesse's Pupil fork on [github](https://github.com/jesseweisberg/pupil). He provides a very well written readme along with demo videos. Or jump straight to his "object detector app" in his [repo](https://github.com/jesseweisberg/pupil/tree/master/pupil_src/shared_modules/object_detector_app).
+
+We hope that others will build on top of Jesse's work and contribute back to his project!
 
-For more information on Jesse Weisberg's plugin, head over to his [github](https://github.com/jesseweisberg/pupil) project.
\ No newline at end of file
+Finally, if you have built something with Pupil, please considering sharing with the community via chat on [DiscordApp](https://pupil-labs.com/chat).
```
Comment (jesseweisberg) [inline contents/articles/2017-12_object_fixation/index.md]: typo: considering -> consider

File: contents/articles/2017-12_object_fixation/index.md (hunk 5 above)
Comment (willpatera) [reply contents/articles/2017-12_object_fixation/index.md]: @jesseweisberg Thanks - this has been added!

**Pull request merged**